src/sentry_value.c

```diff
@@ -1,3 +1,4 @@
+#include "sentry.h"
```

Comment:
Remove this include. this is being pulled in via `sentry_boot.h`, and that should also magically fix your compile error.

Don’t ask me to explain *exactly* whats going on. Just that in C include order matters, and defines that you set have an effect on includes, and `sentry_boot.h` sets some defines that are needed so the Windows SDK does not break on microsofts own compiler. Fun times! Rewrite it in Rust!

Reply:
oh, thanks!